<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>log</class>
 <widget class="QMainWindow" name="log">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SIGNUP</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(fon.jpg);
background-size: cover;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>-15</y>
      <width>471</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 170, 127);
font: 25pt &quot;MV Boli&quot;;</string>
    </property>
    <property name="text">
     <string>Регистрация</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="reg_btn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: rgb(255, 170, 127);
background-color:rgb(255, 85, 0);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border: 1px solid rgb(255, 170, 127);
border-radius: 7px;
width: 200 px;
height: 45px;
}
QPushButton:hover {
background-color: rgba(255, 85, 0, 100);
}
QPushButton:pressed {
background-color: rgba(255, 85, 0, 60);
}</string>
    </property>
    <property name="text">
     <string>Авторизоваться</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_save">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>139</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 170, 127);
background-color: rgb(255, 0, 0);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
border: 1px solid rgb(255, 170, 127);</string>
    </property>
    <property name="placeholderText">
     <string>Имя:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_save">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>300</y>
      <width>139</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 170, 127);
background-color: rgb(255, 0, 0);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
border: 1px solid rgb(255, 170, 127);</string>
    </property>
    <property name="placeholderText">
     <string>Пароль:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
